Source/CarthageKit/Xcode.swift

```diff
@@ -1025,12 +1042,109 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 		}
 }
 
+
+public func getSecuritySigningIdentities() -> SignalProducer<String, CarthageError> {
+	let securityTask = TaskDescription(launchPath: "/usr/bin/security", arguments: ["find-identity", "-v", "-p", "codesigning"])
+	
+	return launchTask(securityTask)
+		|> ignoreTaskData
+		|> mapError { CarthageError.TaskError($0) }
+		|> map { (data: NSData) -> String in
+			return NSString(data: data, encoding: NSStringEncoding(NSUTF8StringEncoding))! as String
+		}
+		|> flatMap(.Merge) { (string: String) -> SignalProducer<String, CarthageError> in
+			return string.linesProducer |> promoteErrors(CarthageError.self)
+		}
+}
+
+public struct CodeSigningIdentity {
```

Comment:
Well no, actually only the `IdentityType` is used (e.g. "iPhone Developer"), since that's what's specified in the project file. The rest aren't used, but does it hurt anything? I could see it being useful/helpful in the future, for presenting more detailed messages to the user.